tests/Unit/ArchTest.php

```diff
@@ -20,7 +20,3 @@
 test('mail should be sent via queue only')
     ->expect('Illuminate\Support\Facades\Mail')
     ->toOnlyBeUsedIn('App\Jobs');
-
-// test('ApiResponser trait should be added to the API controllers')
```

Comment:
Let's add this for now. In case in some project there's any requirement to remove this; then they'll do this action explicitly.

Reply:
We already created a custom test case so that's why I commented.